app/scripts/controllers/deploymentConfig.js

```diff
@@ -74,7 +74,8 @@ angular.module('openshiftConsole')
         var updateHPAWarnings = function() {
             HPAService.getHPAWarnings($scope.deploymentConfig, $scope.autoscalers, limitRanges, project)
                       .then(function(warnings) {
-              $scope.hpaWarnings = warnings;
+              // TODO: this should probably be a $filter we can reuse in other controllers?
```

Comment:
So I can pretty easily eliminate the warning from the controller, but thinking this should be a filter to share with other controllers.